### DISCUSSION TITLE
OnlyBrokenBlocks

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#90] [Ideas]
Новое свойство в классе BuildBlocksSet, которое выдает ВСЕ ТОЛЬКО поломанные блоки.

**Comment by justqstn** [comment]
не имеет смысла

**Comment by kkohno** [comment]
можно пример зачем и как юзать?

**Comment by KukushkaSex** [comment]
Много чего сделать, например ту же битву строителей..

**Comment by forwork10** [comment]
для этого существует палитра.

**Comment by KukushkaSex** [comment]
Как ты ограничишь их, лол

**Comment by forwork10** [comment]
не имеет смысла

**Comment by kkohno** [comment]
> Много чего сделать, например ту же битву строителей..

опишите подробнее, не совсем понятно какой механизм имеется ввиду

**Comment by Kukushechka7-ezx** [comment]
Например игроку даётся задание: построить ту или иную постройку только из сломанных блоков.

**Reply by kkohno** [reply]
ну можете зашить в режим какими блоками строить
